File: Lab 4/C++/main.cpp
#include <iostream>
#include "LetterHeader.h"

using namespace std;
using namespace Artem;

int main()
{
    LetterString value("qwedsadas");

    cout << "Before sorting: " << value.getString() << endl;

    value.sortString();

    cout << "After sorting: " << value.getString() << endl;
    cout << "Length: " << value.getLength() << endl;

    return 0;
}

File: Lab 4/C#/main.cs
using System;
using Artem;

class _Main
{
    static void Main(string[] args)
    {
        LetterString LetterString = new LetterString("dcgfba");
        Console.WriteLine($"Похідний рядок: {LetterString.GetValue()}");

        LetterString.Sort_Str();

        Console.WriteLine($"Відсортований рядок: {LetterString.GetValue()}");
        Console.WriteLine($"Довжина рядка: {LetterString.GetLength()}");
    }
}
